--- pound-2.7/Makefile.in.old	2016-06-07 00:17:55.017105187 +0100
+++ pound-2.7/Makefile.in	2016-06-07 00:20:27.563334597 +0100
@@ -46,10 +46,10 @@
 		${CC} @LDFLAGS@ -o poundctl poundctl.o $(LIBS)
 
 dh512.h:
-		openssl dhparam -5 -C -noout 512 > dh512.h
+		openssl dhparam -5 -C -noout 512 | sed 's/get_dh512()/get_dh512(void)/' > dh512.h
 
 dh@C_DH_LEN@.h:
-		openssl dhparam -5 -C -noout @C_DH_LEN@ > dh@C_DH_LEN@.h
+		openssl dhparam -5 -C -noout @C_DH_LEN@ | sed 's/get_dh@C_DH_LEN@()/get_dh@C_DH_LEN@(void)/' > dh@C_DH_LEN@.h
 
 svc.o:		svc.c dh512.h dh@C_DH_LEN@.h
 		${CC} ${CFLAGS} -c -o svc.o svc.c
--- pound-2.7/http.c.old	2016-06-06 23:15:57.362126782 +0100
+++ pound-2.7/http.c	2016-06-06 23:18:13.992163616 +0100
@@ -82,7 +82,7 @@
         "<html><head><title>Redirect</title></head><body><h1>Redirect</h1><p>You should go to <a href=\"%s\">%s</a></p></body></html>",
         safe_url, safe_url);
     snprintf(rep, sizeof(rep),
-        "HTTP/1.0 %d %s\r\nLocation: %s\r\nContent-Type: text/html\r\nContent-Length: %d\r\n\r\n",
+        "HTTP/1.0 %d %s\r\nLocation: %s\r\nContent-Type: text/html\r\nContent-Length: %zu\r\n\r\n",
         code, code_msg, safe_url, strlen(cont));
     BIO_write(c, rep, strlen(rep));
     BIO_write(c, cont, strlen(cont));
