diff -Naur p/Makefile.in Pound-2.8/Makefile.in
--- p/Makefile.in	2018-05-12 11:43:43.603833381 +0100
+++ Pound-2.8/Makefile.in	2018-05-12 11:46:50.591825726 +0100
@@ -46,10 +46,10 @@
 		${CC} @LDFLAGS@ -o poundctl poundctl.o $(LIBS)
 
 dh512.h:
-		openssl dhparam -5 -C -noout 512 > dh512.h
+		openssl dhparam -5 -C -noout 512 | sed 's/get_dh512()/get_dh512(void)/' > dh512.h
 
 dh@C_DH_LEN@.h:
-		openssl dhparam -5 -C -noout @C_DH_LEN@ > dh@C_DH_LEN@.h
+		openssl dhparam -5 -C -noout @C_DH_LEN@ | sed 's/get_dh@C_DH_LEN@()/get_dh@C_DH_LEN@(void)/' > dh@C_DH_LEN@.h
 
 svc.o:		svc.c dh512.h dh@C_DH_LEN@.h
 		${CC} ${CFLAGS} -c -o svc.o svc.c
